diffkemp/simpll/ModuleComparator.h

```diff
@@ -116,17 +48,22 @@ class ModuleComparator {
     /// DebugInfo class storing results from analysing debug information
     const DebugInfo *DI;
 
+    /// Cache used for dynamic lookup of already compared functions using
+    /// data passed from DiffKemp.
+    Cache *C;
```

Comment:
`C` is definitely not a good name. Use `FunctionCache` or `ResultsCache`. Or, maybe better, call the class `ResultsCache` and the variable `Cache`.

Reply:
The renaming of the class seems like a good idea - it will also make it clear that it caches results and not something else (like macros etc.).